params.go

```diff
@@ -246,6 +251,29 @@ type ParamsContainer interface {
 	GetParams() *Params
 }
 
+type RawParams struct {
+	Params        `form:"*"`
+	APIMode       APIMode `form:"-"`
+	StripeContext string  `form:"-"`
+}
+
+func (p *RawParams) GetAPIMode() APIMode {
+	if p.APIMode == "" {
+		return StandardAPIMode
+	}
+	return p.APIMode
+}
+
+func (p *RawParams) GetStripeContext() string {
+	return p.StripeContext
+}
+
+type RawParamsContainer interface {
```

Comment:
We needed it before to allow users to add request params, but you're right this is no longer necessary since we're switching to the `content` string implementation.